### PR TITLE
[RFC] misc: UNIX => Unix

### PR DESCRIPTION
**Pull request by sethjackson**
See https://github.com/neovim/neovim/pull/3954#discussion_r49894363

### CODE REVIEW

File: src/nvim/if_cscope.c [hunk 1]
```diff
@@ -1841,9 +1841,7 @@ static void sig_handler(int s) {
  */
 static void cs_release_csp(size_t i, int freefnpp)
 {
-  /*
-   * Trying to exit normally (not sure whether it is fit to UNIX cscope
-   */
+   // Trying to exit normally (not sure whether it is fit to Unix cscope)
```
Comment (Pyrohh) [inline src/nvim/if_cscope.c]: The linter is complaining about this line being indented 3 spaces (should be two).

Comment (sethjackson) [inline src/nvim/if_cscope.c]: Oops! Fixed.

**Comment by Pyrohh** [comment]
I just committed #3954, so this needs a rebase to deal with the conflicts (no rush, just a heads up).

**Comment by sethjackson** [comment]
Rebased. Linter errors should all be fixed too now (minus the two where it gets confused about `for`).

**Comment by Pyrohh** [comment]
Thanks @sethjackson, commited, although I tweaked the commit message since, AFAICT, Bell Labs doesn't hold that trademark anymore.

**Pull request closed without merging**